src/SFA.DAS.EAS.Employer_Account.Database/StoredProcedures/GetEmployerAgreement_ByAccountIdAndLegalEntityId.sql

```diff
@@ -0,0 +1,29 @@
+﻿CREATE PROCEDURE [employer_account].[GetEmployerAgreement_ByAccountIdAndLegalEntityId]
```

Comment:
Is this stored proc used?  I can see a repo method (GetLatestAccountLegalEntityAgreement) which calls it but that method has no usages.